RevenueCatUI/Templates/V2/Components/Image/ImageComponentView.swift

```diff
@@ -42,7 +42,12 @@ struct ImageComponentView: View {
                 package: self.packageContext.package
             )
         ) { style in
-            RemoteImage(url: style.url) { (image, size) in
+            RemoteImage(
+                url: style.url,
+                lowResUrl: style.lowResUrl,
+                darkUrl: style.darkUrl,
+                darkLowResUrl: style.darkLowResUrl
+            ) { (image, size) in
```

Comment:
Realized we weren't doing low res for image component and also added dark right away 🤷‍♂️